components/header/components/HeaderNav.tsx

```diff
@@ -17,19 +17,23 @@ const DotNavHeader = (): JSX.Element => {
 };
 
 const DotNavItem = ({ navTitles }: { navTitles: { title: string; id: string }[] }): JSX.Element => {
-    const [active, setActive] = useState('home');
+    const [active, setActive] = useState('documentation');
     return (
         <>
             {navTitles.map((navTitle) => (
                 <li
                     key={navTitle.id}
-                    className={classNames('py-2 px-2 mr-5 font-bold inline-block no-i', {
-                        'text-purple': active === navTitle.id,
-                        'border-b-2': active === navTitle.id,
-                        'border-purple-500': active === navTitle.id
+                    className={classNames('py-2 mr-5 inline-block no-i', {
+                        'font-bold': active === navTitle.id,
```

Comment:
`const isActive = active === navTitle.id,`

```suggestion
                        'font-bold': isActive,
```